server/bundler/utils.js

```diff
@@ -14,6 +14,9 @@ function getAssets( stats ) {
 }
 
 function pathToRegExp( path ) {
+	if ( path === '/' ) {
+		return path;
+	}
 	return new RegExp( '^' + path + '(/.*)?$' );
```

Comment:
that's fine with me @mtias but I hope we add the explanation in a comment to make the intent and "gotchas" clearer